lib/src/impl/realtime/channels.dart

```diff
@@ -26,19 +26,18 @@ class RealtimePlatformChannel extends PlatformObject
 
   RealtimePlatformChannel(this.ably, this.name, this.options) : super() {
     state = spec.ChannelState.initialized;
-    on().listen((ChannelStateChange event) {
+    on().listen((event) {
```

Comment:
I think an expression body `=>` instead of `{}` would be a bit better

Reply:
724714806fcde898167569d3a5dbdcc1efb69bae